packages/app/src/App.tsx

```diff
@@ -90,6 +90,7 @@ const app = createApp({
 
 const routes = (
   <FlatRoutes>
+    <Navigate key="/" to="catalog" replace />
```

Comment:
The main repo example app does `<Route path="/" element={<Navigate to="catalog" />} />` instead - what you have here seems more like an unconditional redirect (the `key` would just be treated as the React key prop right?) unless I'm mistaken